master/Dockerfile.ubuntu

```diff
@@ -10,19 +12,29 @@ MAINTAINER  Buildbot maintainers
 # that everything is rebuilt
 ENV         security_updates_as_of 2016-07-30
 
+# This will make apt-get install without question
+ENV         DEBIAN_FRONTEND noninteractive
```

Comment:
AFAIK in ideal world `DEBIAN_FRONTEND=noninteractive` shouldn't be specified at all: apt/dpkg and their configuration in base image should be smart enough to handle this case properly. But, AFAIK too, the process of fixing this is quite slow.

Though `docker exec -ti bash` + dpkg/apt is rarely needed with BB images it might be used, for example, for debugging.
And instead of introducing potentially new issue by workarounding original issue, I would prefer to limit scope of the workaround to image build process.

I agree with @thaJeztah conclusion https://github.com/docker/docker/issues/4032#issuecomment-163689851:

> @drastik best practice is to not use ENV, because that will make the env-var persist in the final image.

> In most cases, omitting "DEBIAN_FRONTEND" is harmless and only produces some warnings during build (which can be ignored). If really needed, RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q package should work

In the same bug report @markriggins suggested feasible alternative to specifying `DEBIAN_FRONTEND` as env variable https://github.com/docker/docker/issues/4032#issuecomment-192327844: 

> The new ARG directive in the Dockerfile can set this only during the build

> ARG DEBIAN_FRONTEND=noninteractive

I suggest to stick with `ARG DEBIAN_FRONTEND=noninteractive` solution (though personally I prefer to specify `DEBIAN_FRONTEND=noninteractive` only otherwise it doesn't work at all, and mostly it works).